packages/terra-form-input/tests/nightwatch/InputPropPermutations.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Input from '../../lib/Input';
+
+const DefaultInput = () => <Input aria-label="default form input" id="form-input-default" />;
+const InvalidInput = () => <Input aria-label="invalid form input" isInvalid id="form-input-invalid" />;
+const DisabledInput = () => <Input aria-label="disabled form input" disabled id="form-input-disabled" />;
```

Comment:
Add an example for `required`

Reply:
Done c33141d942293cae899fc6e6efcd0587346cc7db